includes/wc-product-functions.php

```diff
@@ -200,6 +200,40 @@ function wc_get_featured_product_ids() {
 	return $featured_product_ids;
 }
 
+/**
+ * Populate a batch of rating count lookup table data for products.
```

Comment:
Please update to `Returns a list of product groups that a specific product belongs to`.